notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/SynchronizedInputsOutputs.scala

```diff
@@ -149,16 +150,16 @@ object SynchronizedInputsOutputs {
     }
 
   /** @return the list of all copied files */
-  def copyFromS3ToHdfs(source: S3Location, target: HdfsLocation, overwrite: Boolean, locationIO: LocationIO): RIO[List[Location]] =
+  def copyFromS3ToHdfs(source: S3Location, target: HdfsLocation, locationIO: LocationIO): RIO[List[Location]] =
     locationIO.isDirectory(source) >>= { isDirectory =>
-      if (!isDirectory) locationIO.copyFile(source, target, overwrite).as(List(source))
+      if (!isDirectory) locationIO.exists(target).flatMap(exist => locationIO.syncFile(source, target).whenM(!exist)).as(List(source))
```

Comment:
Yes, I did this refactoring in 2 steps, but forgot to clean up that code, thanks.